vimeo-networking/src/main/java/com/vimeo/networking/model/uploadquota/Space.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2015 Vimeo
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package com.vimeo.networking.model.uploadquota;
+
+import com.google.gson.annotations.SerializedName;
+import com.vimeo.stag.UseStag;
+
+import org.jetbrains.annotations.NotNull;
+
+import java.io.Serializable;
+
+/**
+ * The representation of the user's remaining upload space. Provides information about how much total space is
+ * available, how much space has been used, and how much is free. Also provides information about whether the data shown
+ * here represents the periodic or lifetime spatial limits imposed on the user, via the {@link Type} enum.
+ */
+@SuppressWarnings("unused")
+@UseStag
+public class Space implements Serializable {
+
+    /**
+     * Signifies the periodic or lifetime nature of this spatial limit.
+     */
+    public enum Type {
+        @SerializedName("periodic")
+        PERIODIC,
+        @SerializedName("lifetime")
+        LIFETIME
+    }
+
+    private static final long serialVersionUID = -1985382617862372889L;
+
+    @SerializedName("free")
+    private long mFree;
+
+    @SerializedName("max")
+    private long mMax;
+
+    @SerializedName("used")
+    private long mUsed;
+
+    @NotNull
+    @SerializedName("showing")
+    private Type mShowing;
```

Comment:
(Optional) The name "showing" probably isn't the best. Are we comfortable breaking away from the API naming convention? Or should we always name our models the same thing as the API?

Pro:
It's easy to look at the API documentation and cross it with the library
Con:
We'll have to inherit their (potentially) legacy names for things - like "clip"

cc: @howardrigberg @msya

Reply:
I would be fine with naming this something else. As you might have guessed, I called the enum `Type` and not `Showing` because it's a very weird name for a property IMO.

Reply:
I also like the idea of calling it something else. `type` or `quotaType` perhaps.

`space.getType()` or `space.getQuotaType()` would be more readable than `space.getShowing()`. I would assume this is okay since API conventions will eventually diverge from platform conventions. As long as we have "serialized name", a consumer should always know how to find true north.

Reply:
Going with `type`